tests/src/common/TestUtils.java

```diff
@@ -48,38 +48,39 @@
 public class TestUtils {
     protected static final Logger logger = Logger.getLogger("basic");
 
-    public static final int SUCCESS_EXIT = 0;
-    public static final int ERROR_EXIT = 1;
-    public static final int MISUSE_EXIT = 2;
-    public static final int ACCEPTED = 202; // 202
-    public static final int BAD_REQUEST = 144; // 400 - 256 = 144
-    public static final int UNAUTHORIZED = 145; // 401 - 256 = 145
-    public static final int FORBIDDEN = 147; // 403 - 256 = 147
-    public static final int NOT_FOUND = 148; // 404 - 256 = 148
-    public static final int NOTALLOWED = 149; // 405 - 256 = 149
-    public static final int CONFLICT = 153; // 409 - 256 = 153
-    public static final int REQUEST_ENTITY_TOO_LARGE = 157; // 413 - 256 = 157
-    public static final int THROTTLED = 173; // 429 (TOO_MANY_REQUESTS) - 256 =
-                                             // 173
-    public static final int TIMEOUT = 246; // 502 (GATEWAY_TIMEOUT) - 256 = 246
-    public static final int DONTCARE_EXIT = -1; // any value is ok
-    public static final int ANY_ERROR_EXIT = -2; // any non-zero value is ok
+    public static final int SUCCESS_EXIT   = 0;
+    public static final int ERROR_EXIT     = 1;
+    public static final int MISUSE_EXIT    = 2;
+    public static final int DONTCARE_EXIT  = -1;  // any value is ok
+    public static final int ANY_ERROR_EXIT = -2;  // any non-zero value is ok
+
+    public static final int ACCEPTED       = 202; // 202
+    public static final int BAD_REQUEST    = 144; // 400 - 256 = 144
+    public static final int UNAUTHORIZED   = 145; // 401 - 256 = 145
+    public static final int FORBIDDEN      = 147; // 403 - 256 = 147
+    public static final int NOT_FOUND      = 148; // 404 - 256 = 148
+    public static final int NOT_ALLOWED    = 149; // 405 - 256 = 149
+    public static final int CONFLICT       = 153; // 409 - 256 = 153
+    public static final int TOO_LARGE      = 157; // 413 - 256 = 157
+    public static final int THROTTLED      = 173; // 429 (TOO_MANY_REQUESTS) - 256 = 173
+    public static final int TIMEOUT        = 246; // 502 (GATEWAY_TIMEOUT) - 256 = 246
 
     private static final File catalogDir = WhiskProperties.getFileRelativeToWhiskHome("catalog");
     private static final File testActionsDir = WhiskProperties.getFileRelativeToWhiskHome("tests/dat/actions");
     private static final File vcapFile = WhiskProperties.getVCAPServicesFile();
     private static final String envServices = System.getenv("VCAP_SERVICES");
+    private static final String loggerLevel = System.getProperty("LOG_LEVEL", Level.WARNING.toString());
 
     static {
-        logger.setLevel(Level.INFO);
+        logger.setLevel(Level.parse(loggerLevel.trim()));
```

Comment:
@domdom82 with the new tests in scala this becomes unnecessarily verbose.
if still desired, for other deployments, you can toggle the value with a configuration in CI.